lib/manager/components/HomeProjectDropdown.js

```diff
@@ -15,88 +17,55 @@ type Props = {
   visibleProjects: Array<Project>
 }
 
-export default class HomeProjectDropdown extends Component<Props> {
+type State = {
+  selectedOption: any
+}
+
+export default class HomeProjectDropdown extends Component<Props, State> {
+  state = {
+    selectedOption: ''
+  }
+
+  handleChange = (selectedOption: any, activeProject: any) => {
+    this.setState({selectedOption})
+    activeProject === null
```

Comment:
I think there is some interference between variables `selectedOption` and `activeProject` in the animated steps below, where when the user clicks an entry after entering a plain `/home/` path, the selection is lost. Any reason to use two variables instead of one?

What I think should happen is, when the user selects an entry from the dropdown, the URL should be updated immediately, as in the current `dev` branch.

![Screen Recording 2021-04-27 at 3 52 26 PM](https://user-images.githubusercontent.com/56846598/116305576-f811ef00-a771-11eb-8b8b-d7b168614d73.gif)